--- vdr-1.4.1/Makefile.fix	2006-08-25 08:05:41.243128500 +0200
+++ vdr-1.4.1/Makefile	2006-08-25 08:17:29.015361500 +0200
@@ -200,30 +200,30 @@
 # VDR binary:
 
 install-bin: vdr
-	@mkdir -p $(BINDIR)
-	@cp vdr runvdr $(BINDIR)
+	@mkdir -p $(DESTDIR)$(BINDIR)
+	@cp vdr runvdr $(DESTDIR)$(BINDIR)
 
 # Configuration files:
 
 install-conf:
-	@if [ ! -d $(VIDEODIR) ]; then\
-	    mkdir -p $(VIDEODIR);\
-	    cp *.conf $(VIDEODIR);\
+	@if [ ! -d $(DESTDIR)$(VIDEODIR) ]; then\
+	    mkdir -p $(DESTDIR)$(VIDEODIR);\
+	    cp *.conf $(DESTDIR)$(VIDEODIR);\
 	    fi
 
 # Documentation:
 
 install-doc:
-	@mkdir -p $(MANDIR)/man1
-	@mkdir -p $(MANDIR)/man5
-	@gzip -c vdr.1 > $(MANDIR)/man1/vdr.1.gz
-	@gzip -c vdr.5 > $(MANDIR)/man5/vdr.5.gz
+	@mkdir -p $(DESTDIR)$(MANDIR)/man1
+	@mkdir -p $(DESTDIR)$(MANDIR)/man5
+	@gzip -c vdr.1 > $(DESTDIR)$(MANDIR)/man1/vdr.1.gz
+	@gzip -c vdr.5 > $(DESTDIR)$(MANDIR)/man5/vdr.5.gz
 
 # Plugins:
 
 install-plugins: plugins
-	@mkdir -p $(PLUGINLIBDIR)
-	@cp $(PLUGINDIR)/lib/lib*-*.so.$(APIVERSION) $(PLUGINLIBDIR)
+	@mkdir -p $(DESTDIR)$(PLUGINLIBDIR)
+	@cp $(PLUGINDIR)/lib/lib*-*.so.$(APIVERSION) $(DESTDIR)$(PLUGINLIBDIR)
 
 # Source documentation:
 
